cluster/ubuntu/util.sh

```diff
@@ -369,7 +369,7 @@ function provision-master() {
   # remote login to MASTER and use sudo to configue k8s master
   ssh $SSH_OPTS -t $MASTER "source ~/kube/util.sh; \
                             groupadd -f -r kube-cert; \
-                            ~/kube/make-ca-cert ${MASTER_IP} IP:${MASTER_IP},IP:${SERVICE_CLUSTER_IP_RANGE%.*}.1,DNS:kubernetes,DNS:kubernetes.default,DNS:kubernetes.default.svc,DNS:kubernetes.default.svc.cluster.local; \
+                            ~/kube/make-ca-cert.sh ${MASTER_IP} IP:${MASTER_IP},IP:${SERVICE_CLUSTER_IP_RANGE%.*}.1,DNS:kubernetes,DNS:kubernetes.default,DNS:kubernetes.default.svc,DNS:kubernetes.default.svc.cluster.local; \
```

Comment:
Its not a duplication. Removing it will cause cert errors when the master starts up. Line#381 refers to the provisioning of a master/minion node. This function refers to provisioning of a master node only. We need to fix the code rather than removing it.